backend/main/shared/dps.go

```diff
@@ -0,0 +1,213 @@
+package shared
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"strings"
+
+	"github.com/onflow/cadence"
+	dpsApi "github.com/onflow/flow-dps/api/dps"
+	"github.com/onflow/flow-dps/codec/zbor"
+	"github.com/onflow/flow-dps/service/invoker"
+	"github.com/onflow/flow-go-sdk"
+	_flow "github.com/onflow/flow-go/model/flow"
```

Comment:
when I saw `_flow` it confused me as I thought it was using underscores for dup param names and types like in cadence. How about naming it `flowGo`? I feel like that way I would straightaway know it means the `flow-go` sdk